db/order_products.js

```diff
@@ -90,6 +117,24 @@ const getOrderProductById = async (orderProductId) => {
     }
 
 }
+//Returns order products with the order id passed
+const getOrderProductsByOrderId = async (orderId) => {
+    console.log("OrderId",orderId)
+    try{
+
+        const { rows: [ orderProduct ] } = await client.query(`
+            SELECT * FROM order_products
+            WHERE "orderId"=$1;
+        `, [orderId])
+        console.log('The orderproduct we need', orderProduct)
```

Comment:
Since this function should return the products inside an order, we should expect an array of products, so I don't think orderProduct should be destructed out of rows here